src/mpl/include/mpl_trmem.h

```diff
@@ -316,7 +316,7 @@ static inline void MPL_Memcpy_stream(void *dest, const void *src, size_t n)
 
 #else
 
-#ifdef HAVE_SSE2
+#ifdef HAVE_MM_STREAM_SI128
```

Comment:
This will be defined in `mpichconf.h`. Will it get included before this header for this to work? Typically we make sure to include `mpl.h` before anything else.

Reply:
I ended up adding a new m4 to deal with the autoheader for generating the correct config.h.in for MPL. Now the pass-through of the definition should work. And the MPL is revert back to use MPL_ prefixed macros.